src/component/utils/getElementForBlockType.js

```diff
@@ -12,32 +12,20 @@
 
 'use strict';
 
+const DefaultDraftBlockRenderMap = require('DefaultDraftBlockRenderMap');
+
 import type {DraftBlockType} from 'DraftBlockType';
+import type {DraftBlockRenderMap} from 'DraftBlockRenderMap';
+
+function getElementForBlockType(
+  blockType: DraftBlockType,
+  customBlockMap: DraftBlockRenderMap
+): string {
+  const draftBlockRenderMap = customBlockMap !== undefined ? customBlockMap : DefaultDraftBlockRenderMap;
```

Comment:
Is it actually necessary to check for `undefined`, given that there is a default prop for `customBlockMap`?